src/components/form/range/range.tsx

```diff
@@ -116,6 +116,23 @@ export class EuiRangeClass extends Component<
     });
   };
 
+  handleAriaValueText = (
+    ticks: EuiRangeTick[],
+    currentVal: string | number
+  ): string | undefined => {
+    const target = ticks.find(
+      (tick) => tick.value.toString() === currentVal.toString()
+    );
+
+    if (target) {
+      return target.accessibleLabel
+        ? `${target.value.toString()}, (${target.accessibleLabel})`
+        : typeof target.label === 'string'
+        ? `${target.value.toString()}, (${target.label})`
```

Comment:
We don't need to `.toString()` value again since it's being called in a template literal, that handles it for us
```suggestion
        ? `${target.value}, (${target.accessibleLabel})`
        : typeof target.label === 'string'
        ? `${target.value}, (${target.label})`
```